CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# 3.7.8
+
+## Deprecation
+
+- Started the deprecation of all functions below with alert on console:
```

Comment:
We don't use the past tense in the changelog.
Maybe it could be re-worded like: `Deprecate all the below functions with warnings in the console.`